src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/SEC101_152_AzureStorageAccountIdentifiableKey.cs

```diff
@@ -16,5 +17,35 @@ public AzureStorageAccountIdentifiableKey()
         public override ISet<string> Signatures => IdentifiableMetadata.AzureStorageSignature.ToSet();
 
         public override IEnumerable<ulong> ChecksumSeeds => new[] { IdentifiableMetadata.AzureStorageAccountChecksumSeed };
+
+        public override IEnumerable<string> GenerateTruePositiveExamples()
+        {
+            string sampleKey = string.Empty;
+
+            foreach (string example in base.GenerateTruePositiveExamples())
+            {
+                sampleKey = example;
+                yield return example;
+            }
+
+            var legacyStorageAccountKey = new AzureStorageAccountLegacyCredentials();
+
+            // The legacy storage account key rule requires more context in the scan contents,
```

Comment:
>legacy [](http://example.com/codeflow?start=19&length=6)

This comment reflects a prominent problem report. An azure storage connection string expresses the key as an unquoted token following the pattern `AccountKey=`. Because we improperly included the equals sign as a forbidden prefixed delimiter, we dropped these finds.

AccountKey=SOMEKEY